docs/project/glossary.md

```diff
@@ -36,9 +36,11 @@ terminology.
 | ProjectN | Codename for the first version of [.NET Native for UWP](https://msdn.microsoft.com/en-us/vstudio/dotnetnative.aspx). |
 | R2R | Ready-to-Run. A flavor of native images - command line switch of [crossgen](../workflow/building/coreclr/crossgen.md). |
 | Redhawk | Codename for experimental minimal managed code runtime that evolved into [CoreRT](https://github.com/dotnet/corert/). |
+| SEH | Structured Exception Handling - Microsoft extension to C see [details](https://docs.microsoft.com/en-us/cpp/cpp/structured-exception-handling-c-cpp) |
 | SOS | [Son of Strike](https://docs.microsoft.com/en-us/archive/blogs/jasonz/sos-debugging-of-the-clr-part-1). The debugging extension for DbgEng based debuggers. Uses the DAC as an abstraction layer for its operation. |
 | SuperPMI | JIT component test framework (super fast JIT testing - it mocks/replays EE in EE-JIT interface) - see [SuperPMI details](https://github.com/dotnet/runtime/blob/main/src/coreclr/ToolBox/superpmi/readme.txt). |
 | SVR | The CLR used to be built as two variants, with one called "mscorsvr.dll", to mean the "server" version. In particular, it contained the server GC implementation, which was intended for multi-threaded apps capable of taking advantage of multiple processors. In the .NET Framework 2 release, the two variants were merged into "mscorwks.dll". The WKS version was the default, however the SVR version remained available. |
+| TFM | [Target Framework Moniker](https://docs.microsoft.com/en-us/dotnet/standard/frameworks) such as `net6.0` or `wpa81`. |
```

Comment:
```suggestion
| TFM | [Target Framework Moniker](https://docs.microsoft.com/en-us/dotnet/standard/frameworks) such as `net6.0` or `netstandard2.0`. |
```
wpa81 is a very obscure TFM. I would be better to use the more common one as an example.